standard.md

```diff
@@ -0,0 +1,272 @@
+1. Change in `description`
+1. Change in `externalDocs`
+1. Change in `tags`
+
+##### `message`
+
+**Breaking:**
+
+1. Change in `contentType`
+1. Change in `name`
+1. Change in `payload`
+1. Change in `traits`
+1. Change in `headers`
+
+**Non-Breaking:**
+
+1. Change in `summary`
+1. Change in `description`
+1. Change in `examples`
+1. Change in `title`
+1. Change in `tags`
+1. Change in `externalDocs`
+1. Change in `correlationId`
+
+#### `parameters`
+
+**Breaking:**
+
+1. Change in `schema`
+1. Change in `location`
+1. Change in value of `$ref`
+
+**Non-Breaking:**
+
+1. Change in `description`
+
+### `components`
+
+Any change in `components` is **non-breaking**.
+
+**Reason:** The parser will inline the `$ref` during the parsing phase.
+
+### `tags`
+
+Any change in `tags` is **non-breaking**.
+
+### `externalDocs`
+
+Any change in `externalDocs` is **non-breaking**.
+
+## Summary
+
+### Breaking changes
+
+1. Change in `major` version in `asycnapi`
+1. Change in `defaultContentType` field
+1. Change in `version` in `info`
+1. Change in `termsOfService` in `info`
+1. Change in `license` field in `info`
+1. Removal of a server(equivalent to change in server name)
+1. Change in `url` in `servers`
+1. Change in `protocol` in `servers`
+1. Change in `protocolVersion` in `servers`
+1. Change in `default` (`variables`) in `servers`
+1. Removal in `enum` (`variables`) in `servers`
+1. Change in `security` in `servers`
+1.  Removal of channel
+1.  Removal of `subscribe` or `publish` field in `channels`
+1.  Change in value of `$ref` in `channels`
+1. Change in `operationId` in `subscribe`|`publish`
+1. Change in `traits` in `subscribe`|`publish`
+1. Change in `contentType` in `message`
+1. Change in `name` in `message`
+1. Change in `payload` in `message`
+1. Change in `traits` in `message`
+1. Change in `headers` in `message`
+1. Change in `schema` in `parameters`
+1. Change in `location` in `parameters`
+1. Change in value of `$ref` in `parameters`
+
+### Non-Breaking changes
+
+1. Change in `minor`/`patch` version in `asyncapi`
+1. Change in `id` field
+1. Change in `title` in `info`
+1. Change in `description` in `info`
+1. Change in `contact` in `info`
+1. Server being added
+1. Change in `description` in `servers`
+1. Change in `description` (`variables`) in `servers`
+1. Change in `examples` (`variables`) in `servers`
+1. Addition in `enum` (`variables`) in `servers`
+1. Change in `description` in `channels`
+1. Addition of `subscribe` or `publish` field in `channels`
+1. Change in `summary` in `subscribe`|`publish`
+1. Change in `description` in `subscribe`|`publish`
+1. Change in `externalDocs` in `subscribe`|`publish`
+1. Change in `tags` in `subscribe`|`publish`
+1. Change in `summary` in `message`
+1. Change in `description` in `message`
+1. Change in `examples` in `message`
+1. Change in `title` in `message`
+1. Change in `tags` in `message`
+1. Change in `externalDocs` in `message`
+1. Change in `correlationId` in `message`
```

Comment:
this one 🤔 

correlationId's purpose is purely for observability, so you know where in the message you can find information about the message id to track it in the system.

Not sure but it might be considered a breaking change from tooling perspective 🤔 

just keep in mind it is an object, so change in `description` is non-breaking, I'm just thinking about `location` property only, change or removal.

The tricky part here is what is the definition of breaking change, is it like breaking change in the API, or breaking change in the AsyncAPI document, I think these are 2 different things

Reply:
> The tricky part here is what is the definition of breaking change, is it like breaking change in the API, or breaking change in the AsyncAPI document, I think these are 2 different things

From the POV of a user, the change in API will be a breaking change (if it affects the way user interacts with the API).

Is changing the `location` in `correlationId` a breaking change? (In terms of API).

Reply:
for me, it is because if you do not notice the change, your observability configuration basically fails and devops is affected.